tests/unit/test_httpchecksum.py

```diff
@@ -166,7 +170,7 @@ def test_request_checksum_algorithm_model_unsupported_algorithm(self):
         operation_model = self._make_operation_model(
             http_checksum={"requestAlgorithmMember": "Algorithm"},
         )
-        params = {"Algorithm": "crc32"}
+        params = {"Algorithm": "sh256"}
```

Comment:
When an empty list is passed to `supported_algorithms`, the custom missing dependency error will be raised if the provided algorithm is `crc32`. This isn't technically true so is this acceptable?

Reply:
This is only ever used in testing, so it won't matter to the end user.

Is `sh256` a typo here?